<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>et_listeAttenteTests</class>
 <widget class="QWidget" name="et_listeAttenteTests">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>811</width>
    <height>676</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="stw_ParamTests">
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="0" column="0">
        <layout class="QHBoxLayout" name="h_Layout_et_InterfaceExecutionTest"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="stw_listeAttenteTest">
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0">
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="button_Annuler">
           <property name="text">
            <string>&amp;Annuler</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="button_ExecuterTests">
           <property name="text">
            <string>&amp;Exécuter les tests</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item row="0" column="0">
        <widget class="QTabWidget" name="tabWidget">
         <property name="currentIndex">
          <number>0</number>
         </property>
         <widget class="QWidget" name="tab_listeTests">
          <attribute name="title">
           <string>Liste des tests prévus</string>
          </attribute>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <pointsize>12</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="text">
              <string>Tests en attente :</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QTableWidget" name="tableWidget_TestsEnAttente">
             <property name="alternatingRowColors">
              <bool>true</bool>
             </property>
             <property name="selectionMode">
              <enum>QAbstractItemView::SingleSelection</enum>
             </property>
             <property name="selectionBehavior">
              <enum>QAbstractItemView::SelectRows</enum>
             </property>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>130</number>
             </attribute>
             <attribute name="horizontalHeaderStretchLastSection">
              <bool>true</bool>
             </attribute>
             <column>
              <property name="text">
               <string>idTest</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Nom Test</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Interface</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Canal</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Date/Heure Début</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Etat du test</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="1" column="1">
            <layout class="QVBoxLayout" name="verticalLayout">
             <item>
              <widget class="QPushButton" name="button_Annuler_2">
               <property name="text">
                <string>&amp;Annuler</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_AjouterTest">
               <property name="text">
                <string>Ajouter un test</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="button_SupprimerTest">
               <property name="text">
                <string>Supprimer un test</string>
               </property>
              </widget>
             </item>
             <item>
              <spacer name="verticalSpacer">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>40</height>
                </size>
               </property>
              </spacer>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
